### PR TITLE
Add Sebastián Ramírez keynote speaker blog

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>

### CODE REVIEW

**Comment by vipulgupta2048** [comment]
Solves https://github.com/pythonindia/inpycon2021-tasks/issues/54

File: content/blog/2021/announcing-keynote-speaker-sebastian.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Title:Announcing our second Keynote Speaker for PyCon India 2021 - Sebastián Ramírez
+Date: 2021-06-02 17:00
+Category: 2021
+Slug: keynote-announcement-sebastián-ramírez
```
Comment (anistark) [inline content/blog/2021/announcing-keynote-speaker-sebastian.md:4]: should we have "á" or "a" for slug?

Comment (anistark) [reply content/blog/2021/announcing-keynote-speaker-sebastian.md:4]: Let's keep all english common characters in url?
`keynote-announcement-sebastian-ramirez` ??

Comment (vipulgupta2048) [reply content/blog/2021/announcing-keynote-speaker-sebastian.md:4]: I was thinking about it at the time of creating the PR, will make the change

**Comment by vipulgupta2048** [comment]
@anistark made the changes, ready to go 🚀

**Review by anistark — APPROVED** [review]

File: content/blog/2021/announcing-keynote-speaker-sebastian.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+Title:Announcing our second Keynote Speaker for PyCon India 2021 - Sebastián Ramírez
+Date: 2021-06-02 17:00
+Category: 2021
+Slug: keynote-announcement-sebastian-ramirez
+Summary: We are happy to announce **Sebastián Ramírez** as a Keynote Speaker for PyCon India 2021!
+
+![Sebastián Ramírez](https://in.pycon.org/2021/assets/images/s2.png)
```
Comment (anistark) [inline content/blog/2021/announcing-keynote-speaker-sebastian.md:7]: image link incorrect. replace `s2.png` with `sebastian.png`

**Review by anistark — CHANGES_REQUESTED** [review]
image link replace

File: content/blog/2021/announcing-keynote-speaker-sebastian.md (hunk 2 above)
Comment (anistark) [reply content/blog/2021/announcing-keynote-speaker-sebastian.md:7]: @vipulgupta2048 ^

Comment (vipulgupta2048) [reply content/blog/2021/announcing-keynote-speaker-sebastian.md]: Yeah, I was flying blind on that one and couldn't find the PyCon Website PR when I wrote this. Good catch. Made the change.

**Comment by vipulgupta2048** [comment]
✔️

**Review by anistark — APPROVED** [review]

**Comment by anistark** [comment]
LGTM. 

@sayanchowdhury @palnabarun Can we merge and deploy this?

**Comment by anistark** [comment]
@vipulgupta2048 The image url has assets. Need to fix that. Can you raise another PR?

https://in.pycon.org/2021/images/sebastian.png is the right image url

**Pull request merged**